migrations.go

```diff
@@ -14,13 +12,11 @@ import (
 
 func CheckMigration() error {
 	stump.Log("checking if repo migration is needed...")
-	p := util.IpfsDir()
-	oldverB, err := ioutil.ReadFile(filepath.Join(p, "version"))
+	oldver, err := GetCurrentVersion()
```

Comment:
actually, now that i think about it. if theres no version file, theres no repo to migrate. So we can just short circuit out of this check in that case.